src/content/docs/enterprise/organization-management/settings/custom-qa-checks.mdx

```diff
@@ -1,48 +1,64 @@
 ---
-title: Custom QA Checks
-description: Configure custom QA checks to ensure high-quality translations
+title: Custom and External QA Checks
+description: Configure custom and external QA checks to ensure high-quality translations
```

Comment:
Let's keep the original title and description. According to the section name in the Enterprise UI